python/jupyterlab-chat/jupyterlab_chat/models.py

```diff
@@ -75,12 +75,15 @@ class NewMessage:
 class User(JupyterUser):
     """ Object representing a user """
 
-    mention_name: Optional[str] = None
-    """ The string to use as mention in chat """
-
     bot: Optional[bool] = None
     """ Boolean identifying if user is a bot """
 
+    @property
+    def mention_name(self) -> str:
+        name: str = self.display_name or self.name or self.username
```

Comment:
sanity-check: any risk that they are all nullish? as in should we add a fallback here?